app/views/root/index.html.erb

```diff
@@ -48,9 +48,6 @@
 
       <section class="services-block" aria-labelledby="services-and-information-label" id="services-and-information">
         <div class="inner-block floated-children">
-          <div class="two-column-heading">
-            <h2 id="services-and-information-label">Services and information</h2>
```

Comment:
Now that we've removed the `h2` we're skipping header levels, which we shouldn't do for accessibility.

So we have a few options:

1. Make the following `h3`s into `h2`s
2. Make the `h2` visually hidden so that it's still picked up by screen readers

Given the original reasoning behind the PR is mainly visual I'd suggest doing 2 and updating the text a bit so we avoid the overloading of the `Services and information` term.